### ISSUE TITLE
encode buffers as hex instead of array of bytes in json tests ?

### ISSUE THREAD

**Issue by rom1504** [roblabla/ProtoDef#66]
`new Buffer("381905","hex")` instead of `new Buffer([0x38,0x19,0x05],"hex")`
Kind of shorter and clearer.

**Comment by Saiv46** [comment]
Close this issue, we have `Buffer.from`

**Comment by rom1504** [comment]
Yes the point of the issue is to do that change. It's not done. Is it ?
